tests/remote_config/test_remote_configuration.py

```diff
@@ -300,22 +300,18 @@ def remote_config_asm_extra_services_available(data):
         interfaces.library.wait_for(remote_config_asm_extra_services_available, timeout=30)
 
     def test_tracer_extra_services(self):
-        """ test """
+        """test"""
 
+        # filter extra services
+        extra_services = []
         for data in interfaces.library.get_data():
             if data["path"] == "/v0.7/config":
                 client_tracer = data["request"]["content"]["client"]["client_tracer"]
                 if "extra_services" in client_tracer:
-                    extra_services = client_tracer["extra_services"]
-
-                    if (
-                        extra_services is not None
-                        and len(extra_services) == 1
-                        and extra_services[0] == "extraVegetables"
-                    ):
-                        return
-
-        raise ValueError("extra_services not found")
+                    extra_services.append(client_tracer["extra_services"])
+        assert self.r_outgoing.status_code == 200
+        assert extra_services, "extra_services not found"
```

Comment:
Be gentle with non-python engineers :) 

```suggestion
        assert len(extra_services) != 0, "extra_services not found"
```